repo/packages/M/minio/0/marathon.json.mustache

```diff
@@ -0,0 +1,68 @@
+{
+   "id":"{{service.name}}",
+   "cpus":{{service.cpus}},
+   "mem":{{service.mem}},
+   "instances":1,
+   "maxLaunchDelaySeconds":36000,
+   "args":[
+      "server",
+      "/export"
+   ],
+   "container":{
+      "type":"DOCKER",
+      "volumes": [
+      {
+        "containerPath": "/export",
+        "hostPath":"{{storage.persistant-volume.host-volume}}/{{service.name}}/data", 
+        "mode": "RW"
+      },
+      {
+              "containerPath": "/root/.minio",
+              "hostPath": "{{storage.persistant-volume.host-volume}}/{{service.name}}/config",
+              "mode": "RW"
+      }
+    ],
+      "docker":{
+         "image":"minio/minio:edge",
```

Comment:
please use the image from `resource.json` : "image":"{{resource.assets.container.docker.minio-docker-RELEASE}}"